core/src/main/java/tachyon/client/BlockOutStream.java

```diff
@@ -236,8 +232,8 @@ public void write(int b) throws IOException {
       throw new IOException("Out of capacity.");
     }
 
-    if (mBuffer.position() >= mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES,
-        Constants.MB)) {
+    if (mBuffer.position()
+        >= mTachyonConf.getBytes(Constants.USER_FILE_BUFFER_BYTES, Constants.MB)) {
```

Comment:
sorry please ignore my previous post. This was done by my eclipse formatter (i didn't touch it)